antiphishing/antiphishing.py

```
@@ -1,357 +1,755 @@
-            "kick": "https://www.beehive.systems/hubfs/Icon%20Packs/Red/footsteps.png",
-            "ban": "https://www.beehive.systems/hubfs/Icon%20Packs/Red/ban.png"
+            "ban": 0xff4545,  # Red
+            "timeout": 0xffd966  # Yellow
         }
 
-        description = descriptions[action]
-        colour = colours[action]
-        thumbnail_url = thumbnail_urls[action]
+        description = descriptions.get(action, "Unknown action configured.")
+        colour = colours.get(action, 0xfffffe)
+        thumbnail_url = thumbnail_urls.get(action, "")
```

Ruff (0.8.2)

Local variable `thumbnail_url` is assigned to but never used

Remove assignment to unused variable `thumbnail_url`

(F841)